src/utils/splitHandlerPathAndName.js

```diff
@@ -23,9 +24,15 @@ export default function splitHandlerPathAndName(handler) {
   // Support nested paths i.e. ./src/somefolder/.handlers/handler.run
   //  path: ./src/somefoler/.handlers/handler
   //  name: run
-  const delimiter = handler.lastIndexOf('.')
-  const path = handler.substr(0, delimiter)
-  const name = handler.substr(delimiter + 1)
+  const postpathSplit = postpath.split('.')
```

Comment:
would something like
```
const [filename, ...moduleNesting] = postpath.split('.')
```

work here?

Reply:
i applied this fix. Quite simpler and easy to understand. Thanks!